table-attributes.md

```diff
@@ -118,7 +129,20 @@ ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=allow';
 
 When the above two attributes are configured at the same time, the Regions belonging to the partition `p` can actually be merged. When the attribute of the partition is reset, the partition `p` inherits the attribute from the table `t`, and the Regions cannot be merged.
 
+<CustomContent platform="tidb">
+
 > **Note:**
 >
 > - When there is now only an attribute of a partition, even if the `merge_option=allow` attribute is configured, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).
```

Comment:
```suggestion
> - When only the partition attribute is used, even if you configure `merge_option=allow`, the partition is still split into multiple Regions by default according to the actual number of partitions. To merge all Regions, you need to [reset the attribute of the table](#usage).
```

Reply:
According to the feature owner @rleungx of this doc, "When there is now only an attribute of a partition" means "When only the attribute of the table `t` is used", so the original translation is misleading.